src/hyperion/experiment_plans/flyscan_xray_centre_plan.py

```diff
@@ -376,13 +367,10 @@ def set_aperture_for_bbox_size(
     bbox_size: list[int] | np.ndarray,
 ):
     # bbox_size is [x,y,z], for i03 we only care about x
-    assert aperture_device.aperture_positions is not None
-
-    new_selected_aperture: SingleAperturePosition = (
-        aperture_device.aperture_positions.MEDIUM
-        if bbox_size[0] < 2
-        else aperture_device.aperture_positions.LARGE
+    new_selected_aperture = (
+        AperturePosition.MEDIUM if bbox_size[0] < 2 else AperturePosition.LARGE
     )
+    gda_name = aperture_device._loaded_positions[new_selected_aperture].GDA_name
```

Comment:
should: why make this non-public if you still want to access it here? imo better either leave it public, or provide a method to get the gda name

Reply:
Yh, you're right